### PR TITLE
Rohit/s3 en 2349 disable certain instrumentations

### PR DESCRIPTION
**Pull request by rohit-kadhe**
# Description

Please include a summary of the changes and the related issue to help is review the PR better and faster.

# Checklist for adding new integration:

- [ ] Defined `APIS` in constants [folder](../src/constants/instrumentation/).
- [ ] Updated `SERVICE_PROVIDERS` in [common.ts](../src/constants/common.ts)
- [ ] Created a folder under [instrumentation](../src/instrumentation/) with the name of the integration with atleast `patch.ts` and `instrumentation.ts` files.
- [ ] Added instrumentation in `allInstrumentations` in [init.ts](../src/init/init.ts) and to the `InstrumentationType` in [types.ts](../src/init/types.ts) files.
- [ ] Added examples for the new integration in the [examples](../src/examples/) folder.
- [ ] Updated [package.json](../package.json) to install new dependencies for devDependencies.
- [ ] Updated the [README.md](../README.md) of [langtrace-typescript-sdk](https://github.com/Scale3-Labs/langtrace-typescript-sdk) to include the new integration in the supported integrations table.
- [ ] Updated the [README.md](https://github.com/Scale3-Labs/langtrace?tab=readme-ov-file#supported-integrations) of Langtrace's [repository](https://github.com/Scale3-Labs/langtrace) to include the new integration in the supported integrations table.
- [ ] Added new integration page to supported integrations in [Langtrace Docs](https://github.com/Scale3-Labs/langtrace-docs)

### CODE REVIEW

File: src/instrumentation/llamaindex/instrumentation.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (darshit-s3) [inline src/instrumentation/llamaindex/instrumentation.ts:1]: remove this line

File: src/instrumentation/openai/patch.ts [hunk 2]
```diff
@@ -368,7 +368,7 @@ export function embeddingsCreate (
       'http.max.retries': originalContext?._client?.maxRetries,
       'http.timeout': originalContext?._client?.timeout,
       'llm.embedding_inputs': JSON.stringify([args[0]?.input]),
-      'llm.encoding.formats': JSON.stringify([args[0]?.encoding_format]),
+      'llm.encoding.formats': args[0]?.encoding_format === undefined ? undefined : JSON.stringify([args[0]?.encoding_format]),
```
Comment (darshit-s3) [inline src/instrumentation/openai/patch.ts:371]: unnecessary condition?

Comment (rohit-kadhe) [reply src/instrumentation/openai/patch.ts:371]: No its needed json.stringify should only happen if that value exists

File: src/instrumentation/llamaindex/instrumentation.ts (hunk 1 above)
Comment (rohit-kadhe) [reply src/instrumentation/llamaindex/instrumentation.ts]: done

**Review by darshit-s3 — APPROVED** [review]

**Pull request merged**